.github/workflows/ci.yaml

```diff
@@ -8,25 +8,23 @@ on:
       - main
 
 jobs:
-  lint:
-    strategy:
-      fail-fast: false
```

Comment:
What's the reasoning behind failing fast on lint errors? Wouldn't it be better to get lint and test errors all at once?

Reply:
Good catch. Will add it.